packages/react/src/oxygen-ui-react-auth-components/SignInButton/sign-in-button.scss

```
@@ -20,7 +20,7 @@
   margin: 16px 0;
 }
 
-.OxygenSignInButtonSocial {
+.OxygenSignInButton-social {
```

Is `MUI` using `--` for variations?

```suggestion
.OxygenSignInButton--social {
```

---

`MUI` uses `-` for variations. 
Ex: They use `MuiButton-contained` class name for `contained` variation. Refer [Mui Button.](https://mui.com/material-ui/react-button/#basic-button)

---

Ack 🙌